src/lib/components/autocompleteAddress/index.tsx

```diff
@@ -175,7 +175,7 @@ const AutoCompleteAddress = ({
             service.getDetails(
               { placeId: firstResult.place_id },
               (newPlace) => {
-                onPlaceChanged(newPlace, false);
+                onPlaceChanged(newPlace ? newPlace : undefined, false);
```

Comment:
I think instead of ternary, TS operator `??` can be used?

Reply:
good point 👍